veracode/submit_artifacts.py

```diff
@@ -390,3 +404,63 @@ def submit_artifacts(  # pylint: disable=too-many-return-statements, too-many-br
         return False
 
     return True
+
+
+@validate
+def is_ready_for_new_build(*, upload_api: UploadAPI):
```

Comment:
This isn't accounting for sandbox vs policy scans

Reply:
Also, please type hint the return

Reply:
Addressed both in latest commit.